wikitongues/wikitongues/config/indexing.cfg

```diff
@@ -26,4 +26,28 @@ wikipedia : WikipediaSpider
 fake : fake_spider
 
 [local_config_file]
-local config file : wikitongues-language-indexing.cfg
+file_name : wikitongues-language-indexing.cfg
+
+[airtable_languages_table]
+# Uncomment below to override
+# table_name : Languages
+# id_column : Identifier
+# base_id : {your base id}
+# api_key : {your api key}
+# fake : true
+
+[airtable_items_table]
+# table_name : External Resources
+# id_column : Url
+# base_id : {your base id}
+# api_key : {your api key}
+# fake : true
+
+[DEFAULT]
+# MODIFY
+base_id : {your base id}
+api_key : {your api key}
+# DO NOT MODIFY
+fake : false
+table_name : Languages
+id_column : Identifier
```

Comment:
There shouldn't be default values for these properties. Instead uncomment the settings for these two properties in each of the two table-specific sections

Reply:
alright, sounds good.